src/lib/utility/platform.js

```diff
@@ -70,7 +70,7 @@ export default class Platform {
       // We'll use the `orientation` property which returns an angole of rotation.
       return (Math.abs(parseInt(window.orientation, 10)) === 90) ? this.orientations.LANDSCAPE : this.orientations.PORTRAIT
     } else {
-      console.error(`Cannot determine orientation of a device, returning a default value of "portrait"`)
+      console.warn(`Alpheios cannot determine the orientation of this device, assuming "portrait"`)
```

Comment:
it seems to me that it is really difficult to see such a console log on a mobile device. May be it is not useful&

Reply:
that's a good point. I will remove it.